complex.c

```diff
@@ -1850,16 +1853,34 @@ nucomp_to_r(VALUE self)
 
 /*
  * call-seq:
- *    cmp.rationalize([eps])  ->  rational
- *
- * Returns the value as a rational if possible (the imaginary part
- * should be exactly zero).
- *
- *    Complex(1.0/3, 0).rationalize  #=> (1/3)
- *    Complex(1, 0.0).rationalize    # RangeError
- *    Complex(1, 2).rationalize      # RangeError
- *
- * See to_r.
+ *   rationalize(epsilon = nil) -> rational
+ *
+ * Returns a Rational object whose value is exactly or approximately
+ * equivalent to that of <tt>self.real</tt>.
+ *
+ * With no argument +epsilon+ given, equivalent to Complex#to_r:
```

Comment:
It's not equivalent to `Complex#to_r`, for example:

```ruby
irb(main):002> Complex(3.14159, 0).rationalize
=> (314159/100000)
irb(main):004> Complex(3.14159, 0).to_r
=> (3537115888337719/1125899906842624)
```

Reply:
Fixed (I think).